src/Coordination/KeeperReconfiguration.cpp

```diff
@@ -79,7 +85,7 @@ String serializeClusterConfig(const ClusterConfigPtr & cfg, const ClusterUpdateA
             new_config.emplace_back(RaftServerConfig{*cfg->get_server(priority->id)});
         }
         else
-            UNREACHABLE();
+            throw Exception(ErrorCodes::UNEXPECTED_ZOOKEEPER_ERROR, "Unexpected update");
```

Comment:
I would also use `chassert` here. It is better to crash keeper than doing something wrong. `UNREACHABLE` makes keeper crash, so do not change that.

Reply:
It's okay, we will abort in one of the catch blocks if it's in critical part of code.

But please use `LOGICAL_ERROR`